tests/test_merge.py

```diff
@@ -41,9 +41,9 @@ def test_merge_from_fixtures(tmp_path):
     and validates against expected network sets, i.e. invalid networks are not merged,
     and subnets are merged into the root network appropriately.
     '''
-    base_nets, base_subnets = __read_test_vectors("tests/data/base_file.csv")
+    base_nets, base_nets_to_exclude = __read_test_vectors("tests/data/base_file.csv")
     base_path = tmp_path / "base.txt"
-    extra_nets, extra_subnets = __read_test_vectors("tests/data/extra_file.csv")
+    extra_nets, extra_nets_to_exclude = __read_test_vectors("tests/data/extra_file.csv")
     extra_path = tmp_path / "extra.txt"
     # write the networks to disk, generating ASNs for each network
     generate_ip_file(base_path, build_file_lines(base_nets, generate_asns(len(base_nets))))
```

Comment:
Not sure if I mentioned this before but as a next step we'll want to make the ASNs part of the fixture files too since they also allow us to test the scenarios where the ASN is relevant for the network being included or not. But let's keep this for a follow-up.